clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -332,6 +332,7 @@ public static EnumSet<ApiKeys> controllerApis() {
     public static EnumSet<ApiKeys> clientApis() {
         List<ApiKeys> apis = Arrays.stream(ApiKeys.values())
             .filter(apiKey -> apiKey.inScope(ApiMessageType.ListenerType.ZK_BROKER) || apiKey.inScope(ApiMessageType.ListenerType.BROKER))
+                .filter(apiKey -> !apiKey.messageType.latestVersionUnstable())
```

Comment:
We can't add this change to `clientApis` since it is used by other callers. Please add the filter into `toHtml`. Also, please have test for it